mta/ext.go

```diff
@@ -202,7 +202,9 @@ func mergeRequires(requires []Requires, extRequiresProvider requiresProvider, un
 func isFieldOverWritable(field string, meta map[string]MetaData, m map[string]interface{}) bool {
 	if meta != nil && m[field] != nil {
 		if metaData, exists := meta[field]; exists {
-			return metaData.OverWritable
+			if metaData.OverWritable != nil {
```

Comment:
Instead of this 'if' maybe add the condition to the previous line
```
if metaData, exists := meta[field]; exists && metaData.OverWritable != nil {
```